sql/conseil.sql

```diff
@@ -41,6 +41,19 @@ BEGIN
 END;
 $$;
 
+-- CREATE PROCEDURE tezos.defer_fork_constraints()
```

Comment:
why is this commented?

Reply:
I just polished the PR. It was in fact a leftover and I removed it.